test/test-server/cli_handler.go

```diff
@@ -0,0 +1,28 @@
+package testserver
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	cliv1 "github.com/confluentinc/ccloud-sdk-go-v2/cli/v1"
+)
+
+// Handler for: /cli/v1/feedbacks
+func handleFeedbacks(t *testing.T) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		req := new(cliv1.CliV1Feedback)
+		err := json.NewDecoder(r.Body).Decode(req)
+		require.NoError(t, err)
+		if len(*req.Content) > 20 {
+			w.WriteHeader(http.StatusForbidden)
+			_, err := io.WriteString(w, `{"errors":[{"status":"403","detail":"feedback exceeds the maximum length"}]}`)
```

Comment:
Can write a struct and serialize it to JSON instead of writing a raw JSON string?